src/main/java/com/adobe/commerce/cif/api/CartApi.java

```diff
@@ -345,6 +368,40 @@ Cart deletePayment(
         @ApiParam(value = ACCEPT_LANGUAGE_DESC)
         @HeaderParam(ACCEPT_LANGUAGE) String acceptLanguage);
 
+    @DELETE
+    @Path("/{id}/payments/{paymentId}")
+    @ApiOperation(value = "Removes the payment with the given ID from the shopping cart.")
+    @ApiResponses(value = {
+        @ApiResponse(code = HTTP_OK, message = HTTP_OK_MESSAGE, response = Cart.class),
+        @ApiResponse(code = HTTP_BAD_REQUEST, message = HTTP_BAD_REQUEST_MESSAGE, response = ErrorResponse.class),
+        @ApiResponse(code = HTTP_FORBIDDEN, message = HTTP_FORBIDDEN_MESSAGE, response = ErrorResponse.class),
+        @ApiResponse(code = HTTP_NOT_FOUND, message = HTTP_NOT_FOUND_MESSAGE, response = ErrorResponse.class)
+    })
+    Cart deletePayments(
```

Comment:
I would like to keep this the same as `postPayments`. I cannot change `postPayments` to singular, since the method signature of it and the deprecated version are the same.

Reply:
Right, I find it more confusing when looking at the `postPayment` and `postPayments` as these ones are the same. Why not using a different name like `deleteCartPayment` or `deletePaymentById`

Reply:
I kind of agree with Sorin about the naming, having plural make it "sound" like it's possible to update or delete multiple payments with one request. What about having `postCartPayment` and `deleteCartPayment` (similar to `postCartEntry` and `deleteCartEntry`)?

Reply:
@masodu @cjelger done.